buidler/tasks/gelato/core/gelato_sys_admin/task.setGelatoGasPriceOracle.js

```diff
@@ -17,10 +17,13 @@ export default task(
       if (taskArgs.log)
         console.log("\n gc-setgelatogaspriceoracle: TaskArgs\n", taskArgs);
 
+      const sysAdmin = getSysAdmin();
+
       const gelatoCore = await run("instantiateContract", {
         contractname: "GelatoCore",
         contractaddress: taskArgs.gelatocoreaddress,
         write: true,
+        signer: sysAdmin,
```

Comment:
You probably do this because you deployed from SysAdmin - but this assumes that we want all these PKs in our .env which is unnecessarily cluttered. 

Fine for now but we probably need to redeploy the setup eventually to make everything simpler.

Reply:
Yes we should maybe refactor to it to only have one key